Doc 7238 documentation for magma (#2309)
couchbase/docs-server: modules/manage/partials/backend-description-section.adoc

[horizontal]

> *Couchstore*:: the regular mechanism suited to lower volumes of data.

Reviewer: ```suggestion
*Couchstore*:: the traditional storage engine better suited to lower volumes of data.
```

> *Magma*:: the latest backend disk-only mechanism for high performance with much higher data volumes.

Reviewer 2: No need to make definitions bold, our CSS does that anyway

Author: The glossary layout doesn't put the titles in bold if it's set to show as horizontal. Might need a CSS adjustment.

Reviewer 2: Oh yes — in a horizontal layout, you are correct. Ignore this one!

> *Magma*:: the latest backend disk-only mechanism for high performance with much higher data volumes.

Reviewer: ```suggestion
*Magma*:: the latest high performance storage engine for much higher data volumes.
```
